rust-toolchain

```diff
@@ -5,6 +5,6 @@
 # to the user in the error, instead of "error: invalid channel name '[toolchain]'".
 
 [toolchain]
-channel = "1.88.0"
+channel = "1.92.0"
```

Comment:
Usually a proper and comprehensive MSRV bump is done first, see: https://github.com/emilk/egui/pull/7579